### PR TITLE
RPF fixes

### PR DESCRIPTION
**Pull request by SaenkoM**
Development auto auth option in config
Pie chart colors
Line chart scaling fix and style improvements
Funding payment data initialization fix and sync warning

### CODE REVIEW

File: src/components/Auth/Auth.js [hunk 1]
```diff
@@ -22,6 +22,14 @@ class Auth extends PureComponent {
 
   static defaultProps = defaultProps
 
+  componentWillMount() {
```
Comment (jarindr) [inline src/components/Auth/Auth.js]: `componentWillMount` is deprecated, move it to didMount or constructer instead

Comment (SaenkoM) [reply src/components/Auth/Auth.js]: @jarindr fixed :)

File: src/components/Auth/Auth.js [hunk 2]
```diff
@@ -22,6 +22,15 @@ class Auth extends PureComponent {
 
   static defaultProps = defaultProps
 
+  constructor(props) {
+    super()
```
Comment (jarindr) [inline src/components/Auth/Auth.js]: you don't need to call `super` here :P

Comment (SaenkoM) [reply src/components/Auth/Auth.js]: @jarindr it's required

Comment (jarindr) [reply src/components/Auth/Auth.js]: it breaks if we remove it ? hmmm...

Comment (SaenkoM) [reply src/components/Auth/Auth.js]: babel throws `ReferenceError: this hasn't been initialised - super() hasn't been called`
probably assertion that `this` will be available for other methods, even if it's not needed in constructor

Comment (jarindr) [reply src/components/Auth/Auth.js]: okay the, thanks for fixing :D !

Comment (SaenkoM) [reply src/components/Auth/Auth.js]: anytime, thanks for review :)

**Pull request merged**